qr_generator/lib/repositories/seed_repository.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:math';
+import 'package:qr_generator/model/seed_model.dart';
+import 'package:qr_generator/utils/api_client.dart';
+
+class SeedRepository {
+  final ApiClient netUtil;
+  SeedRepository({required this.netUtil});
```

Comment:
**suggestion**: make this a `const` constructor

Reply:
Done!